### DISCUSSION TITLE
how do I install the beam CLI?

### DISCUSSION THREAD

**Discussion by cdhanna** [beamable/BeamableProduct#3079] [Unity Help] [answered]
how do I get the Beam CLI?

**Comment by Brandoffsky** [comment, marked as answer]
To get the Beam CLI, you can follow these steps:

1. Refer to the [Getting Started](https://docs.beamable.com/docs/cli-guide-getting-started) guide for information on setting up the Beamable CLI.
2. Use the following command to create a Beamable project in the CLI: `beam init`
3. Verify your current Beamable project in the CLI with: `beam config`
4. Manage access tokens for use in the CLI by using: `beam login`

For more detailed information on Beam CLI commands, you can visit the [Beam CLI Commands](https://docs.beamable.com/docs/cli-command-reference) documentation.

**Reply by cdhanna** [reply]
thanks!
